iac-extensions/kubernetes/src/main/java/org/sonar/iac/kubernetes/visitors/LocationShifter.java

```diff
@@ -242,16 +235,12 @@ public MarkedYamlEngineException shiftMarkedYamlException(InputFileContext input
     return exception;
   }
 
-  private LinesShifting getOrCreateLinesShifting(InputFileContext ctx) {
-    return linesShiftingPerContext.computeIfAbsent(ctx.inputFile.uri(), context -> new LinesShifting());
-  }
-
   /**
    * Store information related to an original line number.
    * The {@link #linesData} Map contain the original line number as the key.
    * The original line length and target line number are stored in the value, as a {@link LineData} object.
    */
-  static class LinesShifting {
+  public static class LinesShifting {
```

Comment:
Consider extracting this class and `LineData` to separate file as this file is quite big and `LinesShifting` is used in `HelmInputFileContext`